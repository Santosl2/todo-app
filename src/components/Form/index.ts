export * from "./Input";
export * from "./MainForm";
export * from "./Button";
