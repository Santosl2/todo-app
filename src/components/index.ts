export * from "./Header";
export * from "./Form";
export * from "./Badge";
export * from "./TextWithBadge";
export * from "./Item";
export * from "./NoResults";
