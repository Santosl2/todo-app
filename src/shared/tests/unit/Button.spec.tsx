import { Button } from "@/components";
import { render, screen } from "@testing-library/react";

describe("ButtonTest", () => {
  it("should render correctly", () => {
    render(
      <Button type="button" disabled={false}>
        Test
      </Button>
    );

    expect(screen.getByText("Test")).toBeInTheDocument();
  });

  it("match snapshot", () => {
    render(
      <Button type="button" disabled={false}>
        Test
      </Button>
    );
    expect(screen.getByText("Test")).toMatchSnapshot();
  });
});
