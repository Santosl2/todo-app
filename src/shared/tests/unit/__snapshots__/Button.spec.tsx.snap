// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonTest match snapshot 1`] = `
<button
  class="flex h-[3.25rem] items-center gap-2 rounded-lg bg-blue-500 px-4 text-sm font-bold text-gray-100 transition-colors hover:bg-blue-300 disabled:cursor-not-allowed disabled:bg-gray-300"
  type="button"
>
  Test
</button>
`;
